model_compiler/tools/install-tensorrt.py

```diff
@@ -38,15 +39,10 @@ def _get_cuda_version():
 
 
 def _get_tensorrt_url():
```

Comment:
- this modify is not forward compatible, because the _BASE_URL has changed between 3.5/3.6 and 3.8.
- why do not use `_get_cuda_version()` for now?

Reply:
it has been modified for forward compatibility

Reply:
can not find the package for ubuntu1604